### PR TITLE
Releasenotes for CAN bus and LoRa

### PR DESCRIPTION
**Pull request by alexanderwachter**
Added release notes for CAN bus and LoRa

### CODE REVIEW

File: doc/releases/release-notes-2.2.rst [hunk 1]
```diff
@@ -171,7 +171,8 @@ Drivers and Sensors
 
 * CAN
 
-  * <TBD>
+  * Support for CAN_2, but no simultaneous use of CAN_1 and CAN_2.
```
Comment (henrikbrixandersen) [inline doc/releases/release-notes-2.2.rst]: STM32 specific?

File: doc/releases/release-notes-2.2.rst [hunk 2]
```diff
@@ -298,7 +302,11 @@ Drivers and Sensors
 Networking
 **********
 
-* <TBD>
+* CAN bus
+  * CANopen protocol support through 3rd party CANopenNode stack.
+  * Added native ISO-TP subsystem.
+  * Introduced CAN-PRIMARY alias.
```
Comment (henrikbrixandersen) [inline doc/releases/release-notes-2.2.rst]: Perhaps mention secondary as well?

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-2.2.rst [hunk 3]
```diff
@@ -248,6 +248,9 @@ Drivers and Sensors
 
   * <TBD>
 
+* LoRa
+  * Added drivers to support LoRaMac-node library.
```
Comment (Mani-Sadhasivam) [inline doc/releases/release-notes-2.2.rst]: I'd do,

```
Added APIs and drivers needed to support LoRa technology by reusing the LoRaMac-node library.
```

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]
Thanks for this @alexanderwachter. Just a minor change.

File: doc/releases/release-notes-2.2.rst (hunk 2 above)
Comment (alexanderwachter) [reply doc/releases/release-notes-2.2.rst]: We don't have a secondary so far.

**Review by Mani-Sadhasivam — APPROVED** [review]
LoRa part LGTM

**Comment by jhedberg** [comment]
@alexanderwachter could you rebase and re-push? The documentation check failure was a bug that's been resolved.

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**